tar/creation_set.c

```diff
@@ -95,7 +95,7 @@ get_format_code(const char *suffix)
 		{ ".7z",	"7zip" },
 		{ ".ar",	"arbsd" },
 		{ ".cpio",	"cpio" },
-		{ ".iso",	"iso9960" },
+		{ ".iso",	"iso" },
```

Comment:
I guess the real correction would be fixing the typo "iso9960" -> "iso9660"

Reply:
Indeed, replacing 
 		{ ".iso",	"iso9960" }, 
with 
                { ".iso",       "iso9660" },
fixed the issue for me.